src/electos/ballotmaker/validate_edf.py

```diff
@@ -1,6 +1,8 @@
 from pathlib import Path
 
-from electos.ballotmaker.constants import NO_ERRORS, NO_FILE
+from electos.ballotmaker.constants import NO_DATA, NO_ERRORS, NO_FILE
+from electos.ballotmaker.read_edf import read_edf
+from genericpath import isfile
```

Comment:
`genericpath` is a stdlib module: move it up to be grouped with `from pathlib...`

`isort` will take care of this when you get around to it.

Reply:
Good catch! VSCode threw in `from genericpath import isfile` and I didn't think too much about it at the time, although `isort` is placing it where it doesn't belong, too.

In fact, I prefer `Path.is_file()` because it's one less import. I made this change globally. Still don't know why isort put that at the bottom of the import list, but not an issue now.